Moonlight macOS/ViewController/StreamFrameViewController.m

```diff
@@ -0,0 +1,123 @@
+//
+//  StreamFrameViewController.m
+//  Moonlight macOS
+//
+//  Created by Felix Kratz on 09.03.18.
+//  Copyright © 2018 Felix Kratz. All rights reserved.
+//
+
+#import "StreamFrameViewController.h"
+#import "VideoDecoderRenderer.h"
+#import "StreamManager.h"
+#import "Control.h"
+#import "Gamepad.h"
+#import "keepAlive.h"
+
+@interface StreamFrameViewController ()
+@end
+
+@implementation StreamFrameViewController {
+    StreamManager *_streamMan;
+    StreamConfiguration *_streamConfig;
+    NSTimer* _timer;
+    ViewController* _origin;
+    struct Gamepad_device* device;
```

Comment:
What uses this `device` variable?

Reply:
This is a leftover from testing, deleted.